README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/dinubs/coolqlcool)
 
-CoolQLCool (CQC) is an open source Graph QL server that allows you to turn websites into a Graph QL api. You can play around with it in [GraphiQL](https://cool.coolql.cool/graphiql).
+CoolQLCool (CQC) is an open source Graph QL server that allows you to turn websites into a Graph QL api. You can play around with it in [GraphiQL](https://coolql.cool/graphiql).
```

Comment:
haha thanks, I should note that this does point to the same place. Probably better to use the same url everywhere though.